scripts/tools.sh

```diff
@@ -30,6 +30,15 @@ function install_helm() {
     log "INFO" "Helm installation complete. Installed version: $(helm version --short)"
 }
 
+function ensure_hypershift_installed() {
```

Comment:
The issue is that sometimes we need to update hypershift binary as it was changed and old one can fail.
How we should handle this case?